Tools/_sunrise/pr_check_summary.py

```
@@ -0,0 +1,726 @@
+
+            if len(candidate) > COMMENT_SOFT_LIMIT:
+                omitted = len(failed_rows) - index
+                break
+
+            body = candidate
+
+        if omitted:
+            body += f"\n\n_Ещё {omitted} падений не поместились в комментарий. Полный вывод доступен по ссылкам в таблице._"
+
+        return body.strip() + "\n"
+    else:
```

Ruff (0.15.11)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)